Source/WebCore/page/scrolling/ScrollingStateTree.cpp

```diff
@@ -367,11 +367,12 @@ void ScrollingStateTree::traverse(const ScrollingStateNode& node, NOESCAPE const
 
 bool ScrollingStateTree::isValid() const
 {
+    RefPtr rootStateNode = m_rootStateNode;
     if (!m_rootStateNode)
```

Comment:
Would be nice to check `rootStateNode` instead of `m_rootStateNode` here now.

Reply:
Very true, will fix.